src/reference/features/overview/sorting.md

```diff
@@ -0,0 +1,285 @@
+# Sorting by Properties
+
+Throughout Caido - the ability to sort items based on certain properties is integrated. These properties are grouped by category and their availability is dependent on the Caido feature being utilized.
+
+These categories are represented as column names spanning horizontally located along the top row on any table element within Caido.
+
+Clicking a column name will toggle the associated sorting mechanism. Sorting varies in how it is accomplished - ascending/descending order, alphabetical order, numerical order, time-based order and the presence/absence of a property are all factors utilized to provide Caido users with the functionality to easily navigate through data.
+
+<img alt="HTTP History." src="/_images/sorting_example.png" center/>
+
+## Available Sorting Properties
```

Comment:
I created a FAQ that covers these questions. I originally did think about sorting per tool but it may be more redundant having each type copy & pasted multiple times.

But still can do per tool - just let me know!

Reply:
At the same time I am not sure it is viable to have all the fields in one document. What are we expecting users to want from it? I am not sure a flat list is that useful.

Reply:
Just scrap it then?

Reply:
Maybe add the questions/answers to the FAQ?